src/main/java/com/google/testing/compile/CompilationRule.java

```diff
@@ -81,7 +81,7 @@ public Elements getElements() {
    * @throws IllegalStateException if this method is invoked outside the execution of the rule.
    */
   public Types getTypes() {
-    checkState(elements != null, "Not running within the rule");
```

Comment:
This isn't technically wrong, since they're both set together. 

Was this primarily a style change?

Reply:
Yeah basically :)